async-jms/solutions/async-jms/src/main/java/com/redhat/training/messaging/AMQPRouteBuilder.java

```diff
@@ -0,0 +1,20 @@
+package com.redhat.training.messaging;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.springframework.stereotype.Component;
+
+
+@Component
+public class AMQPRouteBuilder extends RouteBuilder {
+
+	@Override
+	public void configure() throws Exception {
+
+		// TODO receive messages from AMQP_Queue and send  to the Log_Orders queue
```

Comment:
Please use always the same format for the TODO comments: `// TODO ` or `// TODO: ` but not both in the same app.